WooCommerce/WooCommerceTests/ViewRelated/Products/Filters/FilterProductListViewModelTests.swift

```diff
@@ -0,0 +1,26 @@
+import XCTest
+@testable import WooCommerce
+
+final class FilterProductListViewModelTests: XCTestCase {
```

Comment:
It looks like good coverage for `FilterProductListViewModel`. We can also add an additional test for the calculated value of `numberOfActiveFilters`. The tests only use `0` and `3`. We can add a test for `2` or `1`. 🙂

Reply:
we have unit tests on `numberOfActiveFilters` before: https://github.com/woocommerce/woocommerce-ios/blob/87760dc3e2df9c09182ef117a1111cf0241526f7/WooCommerce/WooCommerceTests/ViewRelated/Products/Filters/FilterProductListViewModel+numberOfActiveFiltersTests.swift

for `FilterProductListViewModel`, it's not impossible to test modifying the value of each filter type in `filterTypeViewModels` since each `FilterTypeViewModel`'s `selectedValue` is mutable. However, because `FilterTypeViewModel`'s data type is a protocol we have to assume the order of the filter types in `FilterProductListViewModel` (stock status, product status, product type) or do some casting and I thought it's easier to test the `numberOfActiveFilters` given `[FilterTypeViewModel]` like the `numberOfActiveFiltersTests` tests above. do you think there it's worth testing modifying the selected value of `FilterProductListViewModel`'s `filterTypeViewModels`?

Reply:
Oh, I must have forgotten that `FilterProductListViewModel+numberOfActiveFilterTests` exists. That's good enough for me.